app-backend/database/src/main/scala/com/azavea/rf/database/tables/AOIs.scala

```diff
@@ -52,6 +54,41 @@ object AOIs extends TableQuery(tag => new AOIs(tag)) with LazyLogging {
       new OrganizationFkSort(identity),
       new TimestampSort(identity))
 
+  /** Paginate a `list` result. */
+  def page(pageRequest: PageRequest, aois: TableQuery): TableQuery = {
+    aois.sort(pageRequest.sort)
+      .drop(pageRequest.offset * pageRequest.limit)
+      .take(pageRequest.limit)
+  }
+
+  /** Yield a paginated list of all [[AOI]]s that a user has access to. */
+  def listAOIs(
+    pageRequest: PageRequest,
+    queryParams: AoiQueryParameters,
+    user: User
+  )(implicit database: DB): Future[PaginatedResponse[AOI]] = {
+    // TODO: filterUservisibility
```

Comment:
The `filterUserVisibility` method wasn't available to my table and I'm not sure why. Is is necessary?